### ISSUE TITLE
[New resource] OKRs

### ISSUE THREAD

**Issue by samuellfarias** [weDevsOfficial/wp-project-manager#436]
Okr is an indispensable methodology for project management.  The possibility of creating objectives and also Key Results would be very good.

Key Results with start value field and end value field.  Included in a objective.

**Comment by usman244718** [comment]
Yes, I am available. Please share some details about the task. I am eager to collaborate with you. with 6+ years experience, please dm me if you want my portfolio and resume.
[#fullstack](https://twitter.com/hashtag/fullstack?src=hashtag_click) development, [#graphic](https://twitter.com/hashtag/graphic?src=hashtag_click) designing
